lib/galaxy/webapps/galaxy/services/sharable.py

```diff
@@ -134,12 +134,14 @@ def _get_users(self, trans, emails_or_ids: Optional[List] = None) -> Tuple[Set[U
                 try:
                     decoded_user_id = trans.security.decode_id(email_or_id)
                     send_to_user = self.manager.user_manager.by_id(decoded_user_id)
-                    if send_to_user.deleted:
+                    if send_to_user and send_to_user.deleted:
```

Comment:
Are these changes supposed to be in this PR? Or eager `git add` ? :D

Reply:
And I was wondering why I had to fix that again 🤣